test-plan.md

```diff
@@ -93,6 +93,38 @@ The easist way to verify that a project was successfully loaded is to open a .cs
 * In a project that uses globbing (.NET Core), use the VS Code file explorer to add a new file next to the csproj. Intellisense/sighelp/etc should be available in the new file
 * Add a new file and reference a type in it from a different file. Deleting from disk the file containing the referenced type  should produce error messages
 
+#### Razor
+The Razor experience is available when you open a .cshtml file in a valid OmniSharp project.
+
+##### C# Completion
+* Typing `@DateTime.Now` and `@(DateTime.Now)` provides completions throughout typing.
+* Completion is available for types that exist in the project (i.e. `Program`)
+* Typing `@model DateTime` prompts for completion for the `model` symbol and the `DateTime` symbol.
+
+#### C# Signature Help
+* Typing `@Html.Raw()` prompts for signature help inside of the `()`.
+
+#### C# Diagnostics
+* Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file. NOTE: This error squiggly will be misaligned due to known issues.
+
+#### Razor Options
+
+##### razor.disabled
+This option can be set to `true` to disable the above described C# experience.
+
+##### razor.languageServer.directory
+Setting this path to any of the values as listed below, should start the Razor Language server.
```

Comment:
What do you mean by "start the razor language server"? Probably we just want to say here - There should or should not be a razor channel with some expected contents(assuming that is the visual hint to show that the razor server started).